app/Http/Controllers/Api/V1/CurrikiGo/OutcomeController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\API\V1\CurrikiGo;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\V1\CurrikiGo\GetStudentResultRequest;
+use App\Http\Resources\V1\CurrikiGo\StudentResultResource;
+use Illuminate\Http\Request;
+use App\Services\LearnerRecordStoreService;
+
+/**
+ * @group 15. CurrikiGo Outcome
+ *
+ * APIs for generating outcomes against students' submissions.
+ */
+class OutcomeController extends Controller
+{
+    /**
+     * Get Student Results Summary
+     *
+     * Fetch LRS statements based on parameters, and generate a student result summary
+     *
+     * @param GetStudentResultRequest $studentResultRequest
+     *
+     * @responseFile responses/outcome/student-result-summary.json
+     *
+     * @response 500 {
+     *   "errors": [
+     *     "No results found."
+     *   ]
+     * }
+     *
+     * @param GetStudentResultRequest $studentResultRequest
+     * @return Response
+     */
+    public function getStudentResultSummary(GetStudentResultRequest $studentResultRequest)
+    {
+        $data = $studentResultRequest->validated();
+        $response = [];
+        try {
+            $service = new LearnerRecordStoreService();
+            
+            $completed = $service->getCompletedStatements($data);
+           
+            if (count($completed) > 0) {
+                // Assume that this statement already has a result
+                $answers = $service->getAnswersStatementsWithResults($data);
+                if ($answers) {
+                    foreach($answers as $record) {
```

Comment:
Add space after foreach.

Reply:
Have corrected it now, and fixed in a couple other places.